src/client/components/Navigation/Topnav.js

```diff
@@ -119,7 +119,7 @@ class Topnav extends React.Component {
             </Tooltip>
           </Menu.Item>
           <Menu.Item key="user" className="Topnav__item-user">
-            <Link className="Topnav__user" to={`/@${username}`}>
+            <Link className="Topnav__user" to={`/@${username}`} onClick={window.scrollTo(0, 0)}>
```

Comment:
We usually create `onClick` handlers for this sort of thing, to provide it with some context.